frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -898,6 +909,17 @@ function ReaderFooter:addToMainMenu(menu_items)
                     self:refreshFooter(true, true)
                 end,
             },
+            {
+                text = _("Hide empty items"),
+                help_text = _([[This will hide an item from the status bar if it were to display 0 or an off state.]]),
```

Comment:
Maybe something short and to the point like:

This will hide values like 0 or off.